### PR TITLE
Update/file api

### PR DESCRIPTION
**Pull request by inwookie**
# Description (개요) 

File API가 변경 되었는데 업데이트 되어있지 않아서 수정 했습니다. 그리고 issue에 올라온 pem file path 관련해서도 같이 resolve 했습니다. 
https://github.com/smartmind-team/thanosql-docs/issues/173

## Keep in Mind

If you are creating a new statement, your first commit should always be a copy of the statement(model, ThanoSQL, function) template. This will allow reviewers to see what has been changed from the template, making the review process much more efficient.

## Type of change (작업사항)

- [x] REST API description fix 

Following templates can be found from the [Tech-wiki Docs](https://github.com/smartmind-team/tech-wiki/tree/main/project/thanosql/docs). 

## Writing Convention

Make sure to check out the [Writing Convention](https://github.com/smartmind-team/tech-wiki/blob/main/project/thanosql/docs/writing_convention.md). 

## Review Process

Review should be conducted by at least two people.

### CODE REVIEW

**Comment by inwookie** [comment]
@yeemh 

이거 hot-fix로 올라가면 좋을거 같아요

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+You can use ThanoSQL's REST API to remotely send and upload files to your ThanoSQL storage and insert them into any of your table within the database.
+
+!!! warning "__Warning__"
+    - File API supports image, audio, and video data with the following extensions:
+        - Image:  "jpg", "png", "jpeg"
+        - Audio: "mp3", "wav
+        - Video: "mp4", "wmv", "avi"
+
+## File Upload
```
Comment (yeemh) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md:15]:
```suggestion
## __File Upload__
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    ```shell
+    curl -X 'POST' \
+      'https://engine.thanosql.ai/api/v1/file/upload/' \
+      -H 'accept: application/json' \
+      -H 'Authorization: Bearer Issued_API_TOKEN' \
+      -H 'Content-Type: multipart/form-data' \
+      -F 'file=@Data File Path;type=file_type/Data File Type'
+    ```
+
+## File Upload & Insert
```
Comment (yeemh) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md:50]:
```suggestion
## __File Upload & Insert__
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    ```shell
+    curl -X 'POST' \
+      'https://engine.thanosql.ai/api/v1/file/upload/?db_commit=True&table_name=Table name&column_name=Column Name' \
+      -H 'accept: application/json' \
+      -H 'Authorization: Bearer Issued_API_TOKEN' \
+      -H 'Content-Type: multipart/form-data' \
+      -F 'file=@Data File Path;type=file_type/Data File Type'
+    ```
+
+!!! faq "FAQ"
```
Comment (yeemh) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md:89]:
```suggestion
!!! faq "__FAQ__"
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    - In order to use a file within the Jupyter workspace, you must put '/'home/jovyan' in front of the path. 
+
+
+# __How to Delete a File From the ThanoSQL Workspace Database__
+
+You can use ThanoSQL's REST API to delete files from your ThanoSQL storage and remove all rows from any of your tables within the database that have the same value as the file paths.
+
+## File Delete
```
Comment (yeemh) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md:97]:
```suggestion
## __File Delete__
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+    curl -X 'POST' \
+      'https://engine.thanosql.ai/api/v1/file/delete/' \
+      -H 'accept: application/json' \
+      -H 'Authorization: Bearer Issued_API_TOKEN' \
+      -H 'Content-Type: application/json' \
+      -d '{"file_path": "Data File Path"}'
+    ```
+
+
+## File Delete & Remove
```
Comment (yeemh) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md:137]:
```suggestion
## __File Delete & Remove__
```

**Review by yeemh — CHANGES_REQUESTED** [review]
rest_api_thanosql_file.md
파일을 나누거나 heading 2로 나눠주세요.

제목 bold 처리 한글에도 적용해주세요.

File: docs/en/how-to_guides/reference.md [hunk 6]
```diff
@@ -44,5 +44,5 @@ Reference page provides a formal overview of each topic. More information on usi
 ### __ThanoSQL Rest API Reference__
 
 - [__How to Query Using ThanoSQL__](/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_query/)
-- [__How to Upload & Insert to the ThanoSQL Workspace__](/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_upload/)
+- [__How to Upload & Delete a File From the ThanoSQL Workspace Database__](/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file/)
```
Comment (yeemh) [inline docs/en/how-to_guides/reference.md:47]:
```suggestion
- [__How to Upload & Delete a File from the ThanoSQL Workspace Database__](/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file/)
```

File: docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+      -H 'Authorization: Bearer Issued_API_TOKEN' \
+      -H 'Content-Type: multipart/form-data' \
+      -F 'file=@Data File Path;type=file_type/Data File Type'
+    ```
+
+!!! faq "__FAQ__"
+    - In order to use a file within the Jupyter workspace, you must put '/'home/jovyan' in front of the path. 
+
+
+## __How to Delete a File From the ThanoSQL Workspace Database__
```
Comment (yeemh) [inline docs/en/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md:94]:
```suggestion
## __How to Delete a File from the ThanoSQL Workspace Database__
```

File: docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+!!! faq "__FAQ__"
+    - Jupyter 내부의 파일을 사용하려면 경로 앞에 '/'home/jovyan'을 붙여야 합니다.
+
+
+
+## __ThanoSQL 워크스페이스 데이터베이스 파일 삭제__
+
+REST API를 사용하여 자신의 ThanoSQL 저장 공간에 원격으로 파일을 삭제하고 데이터베이스 테이블에 파일 경로가 담긴 행이 있다면 함께 삭제할 수 있습니다.
+
+## __파일 삭제__
```
Comment (yeemh) [inline docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_file.md:99]:
```suggestion
### __파일 삭제__
```

**Review by yeemh — APPROVED** [review]
from이랑 ###만 바꾸고 merge 하시면 될 거 같습니다.

**Comment by yeemh** [comment]
close #173

**Pull request merged**